tools/sync/opengrok.py

```diff
@@ -43,21 +43,37 @@ def delete(logger, uri, params=None):
         return None
 
 
-def post(logger, uri, data=None):
+def post(logger, uri, headers=None, data=None):
+    rv = None
     try:
-        return requests.post(uri, data=data)
+        if headers:
+            s = requests.Session()
```

Comment:
why? can't you pass the headers directly? This doesn't seem neccessary to me

Reply:
yes, as mentioned in the pull req comment the version of `requests` I am working with does not have the param yet.

Reply:
I looked in the doc and it is there http://docs.python-requests.org/en/v1.0.4/user/quickstart/ and the 1.0.4 is from the 2012, which kind of version do you use?

Reply:
Hmm, I tried again and it works, not sure where I tested this originally. Let me rewrite it.